storage/sql.go

```diff
@@ -79,6 +72,17 @@ func newSQL(dial gorm.Dialector, level logger.LogLevel) *SQL {
 		panic(err)
 	}
 
+	if config.Platform() == config.Staging && config.Playground().ForceMigration {
+		// Delete v1 tables for v2 staging
+		_ = db.Migrator().DropTable("users")
+		_ = db.Migrator().DropTable("projects")
+		_ = db.Migrator().DropTable("accounts")
+		_ = db.Migrator().DropTable("transaction_templates")
+		_ = db.Migrator().DropTable("script_templates")
+		_ = db.Migrator().DropTable("transaction_executions")
+		_ = db.Migrator().DropTable("script_executions")
+	}
+
 	migrate(db)
```

Comment:
I feel migrate should also be enclosed within the if above, on prod we don't want to migrate at all I would say, at least not implicitly. If there will be small changes we would wish to just with this function then we could check only the ForceMigration is true.

Reply:
Yeah makes sense